modules/classifications/genesis/genesis.go

```diff
@@ -23,7 +25,25 @@ func (genesisState genesisState) Default() helpers.GenesisState {
 	return genesisState
 }
 
-func (genesisState genesisState) Validate() error { return nil }
+
+func (genesisState genesisState) Validate() error {
+
+	for _, classification := range genesisState.ClassificationList {
+		if errs := validator.Validate(classification); errs != nil {
+			return errs
+		}
+
+		if classification.GetID() == nil { return constants.EntityNotFound}
```

Comment:
validator checks for these conditions, unnecessar

Reply:
Done.